src/Battlescape/AIModule.h

```diff
@@ -139,6 +141,29 @@ class AIModule
 	BattleUnit* getTarget();
 	/// Frees up the destination node for another Unit to select
 	void freePatrolTarget();
+	/// Checks whether anyone on our team can see the target
+	bool visibleToAnyFriend(BattleUnit *target, bool ignoreMyself = false);
+	/// Handles behavior of brutalAI
+	void brutalThink(BattleAction *action);
+	/// Like selectSpottedUnitForSniper but works for everyone
+	bool brutalSelectSpottedUnitForSniper();
+	/// checks whether a position should be avoided
+	bool isAvoidPosition(std::vector<Position> trajectory, Position pos);
```

Comment:
You mean it doesn't avoid duplicates? It shouldn't be unlimited though since it's a locale variable. But I don't think this makes a lot of a difference and I can just remove it. It dates back to when I sent a dedicated spotter while the other aliens assumed a sniping-position. If removing it causes the issue too much, I'll think of a better solution.